pdfjs.config

```diff
@@ -1,6 +1,6 @@
 {
   "betaVersion": "1.2.109",
   "stableVersion": "1.1.469",
-  "baseVersion": "2096a2a94a1f4971a15317bd75d4ec08c7f78f7f",
-  "versionPrefix": "1.2."
+  "baseVersion": "194994a2898a6d01c434d6d3270357799f3ac820",
```

Comment:
So we want to update this to point to commit 467c6f93b5929b7b2b824d5bb794d153544c3620 instead, since that fixes a regression from PR #6551?